python/ittapi/compat/README.md

```diff
@@ -0,0 +1,54 @@
+# ittapi.compat
+
+The module is a wrapper to an earlier implementation of ITT API: (https://github.com/oleksandr-pavlyk/itt-python/tree/master). The idea is to allow users to effortlessly migrate to the `ittapi` package without losing any existing functionality. The module is a wrapper to following ITT APIs:
+
+       - Collection Control API
+       - Domain API
+       - Task API
+       - Process Trace Region API
+
+
+## Usage
+
+### ittapi.compat module
+
+#### Collection Control API
+ 
+The `pause` and `resume` enables to focus the collection on a specific section of code while profiling, and start the application run with collection paused. 
+
+The `detach` detaches the data collection. Application continues to work but no data is collected while profiling. 
+
+       import ittapi.compat as itt
```

Comment:
Let's use python code styled text for such instructions. I mean triple backticks - ```python

Reply:
I have changed the code to python code styled text. Please have a look.